taskchampion/src/server/cloud/server.rs

```diff
@@ -545,11 +545,7 @@ mod tests {
     impl std::fmt::Debug for MockService {
         fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
             f.debug_map()
-                .entries(
-                    self.0
-                        .iter()
-                        .map(|(k, v)| (std::str::from_utf8(k).unwrap(), v)),
-                )
+                .entries(self.0.iter().map(|(k, v)| (k, v)))
```

Comment:
Surely this `map` is superfluous now? Perhaps the whole `impl`?

Reply:
The map is! The impl isn't used but is useful for dbg!ging. Since it's test-only, I don't think there's harm in keeping it.

Reply:
The map is! The impl isn't used but is useful for dbg!ging. Since it's test-only, I don't think there's harm in keeping it.